src/tests/Dragons.test.jsx

```diff
@@ -0,0 +1,69 @@
+import { render, screen } from '@testing-library/react';
+// Component
+import { Provider } from 'react-redux';
+import Dragons from '../components/Dragons/Dragons';
+
+// Sotre
+import store from '../redux/configureStore';
+
+// Reducer and Actions
+import reducer, {
+  fetchDragons,
+  reserveDragon,
+  cancelReserveDragon,
+} from '../redux/dragons/dragons';
```

Comment:
- [ ] Kindly check this import `../redux/dragons/dragons`, I had issues starting your project because the name of the folder starts with a capital `R`edux and you imported it using a Lowercase `r`edux